owners/edf-methanesat.jsonnet

```diff
@@ -0,0 +1,12 @@
+{
+  "contactDisplay": "support@methanesat.org",
+  "contactLink": "https://www.methanesat.org/contact",
+  "description": "Launched into orbit on March 4th, 2024, MethaneSAT, a wholly owned subsidiary of Environmental Defense Fund, is the first satellite developed and funded by an environmental nonprofit organization. It is the only methane-detecting satellite that sees the whole picture, measuring methane emissions from millions of small sources around the world that are a huge part of the problem.
```

Comment:
MethaneSAT is a satellite. It's MethaneSAT, LLC that's the EDF subsidiary. This phrase conflates the two.

Reply:
done.  moved files to new location, will resolve this and we can continue over there.